schemas/content/product.schema.json

```diff
@@ -73,7 +91,13 @@
         "xdm:fabrication": {
```

Comment:
May be this is the same reason as mentioned above for the `schema:description` property. Maybe we should treat this the same by deprecating this property and creating a new property.

I could also imagine that `xdm:fabric` could reflect the intent of this property.

But it might be best to leave this to native english speakers as I am also just another one of those german-natives ;-)